nextcloudappstore/core/feeds.py

```diff
@@ -13,14 +15,28 @@ class AppReleaseRssFeed(Feed):
     description = _('Get the newest app release updates')
     link = reverse_lazy('home')
 
+    def __call__(self, request, *args, **kwargs):
+        self.request = request
+        return super().__call__(request, *args, **kwargs)
+
     def items(self):
-        return AppRelease.objects.order_by('-last_modified')[:10]
+        queryset = AppRelease.objects.order_by('-last_modified')
+        if 'nightly' not in self.request.GET:
```

Comment:
Yeah, but i dont think there's any particular big difference. You could also use ?nightly&prelrease